framework/src/play/db/Evolutions.java

```diff
@@ -392,6 +392,10 @@ private boolean isDisabled() {
         return "false".equals(Play.configuration.getProperty("evolutions.enabled", "true"));
     }
 
+    private static boolean isDatabaseEvolutionDisabled(String dbName) {
```

Comment:
You are doing double-negotiation here: "false" -> "Disabled" -> "!Disabled". The code may be slightly simplified.

Reply:
Of course, in fact I just kept the existing "style" to stay coherent with other methods :D
Can be changed if really required ;)

Reply:
I am not sure. Probably not such important. :)